apps/api/src/server/migration/migrator.service.js

```diff
@@ -38,16 +38,12 @@ export const getMigratorForModel = (modelType) => {
 };
 
 // the mappings are lazily initialised to improve test performance
-const initializeMapping = () => {
-	const schemas = readdirSync('./src/message-schemas/events')
-		.filter((file) => file.endsWith('.schema.json'))
-		.map((file) => {
-			return JSON.parse(readFileSync(`./src/message-schemas/events/${file}`).toString());
-		});
+const initializeMapping = async () => {
```

Comment:
Do we need a change somewhere to await this call? Wasn't async before.

Reply:
Good spot, that's fixed.